### DISCUSSION TITLE
QtCipherSqlitePlugin

### DISCUSSION THREAD

**Discussion by bzctoons** [pawelsalawa/sqlitestudio#3495] [Q&A]
### Details
Hello I Created a database with : https://github.com/devbean/QtCipherSqlitePlugin
Which is based on wxSqlite but I can't open it with sqlitestudio; Any clue ?
Thanks

**Comment by pawelsalawa** [comment]
It's difficult to tell. I assume that password is equal?
Maybe there were some major differences in more recent wxSqlite? (SQLiteStudio uses wxSQLite from a year ago, or even older).

**Comment by bzctoons** [comment]
Hi,
How may I help ?
Is there any way to get more logs within the application ?

**Comment by pawelsalawa** [comment]
What does the status field show when you try to open this database? Is there a message like:
"Could not open database: ...." ?

I'm afraid that the error message (like above) is - by mistake - ignored and not printed to the user when you add new database. It is properly displayed later if DB was already added and there is error re-opening it.

To walkaround this issue I figured out following way:

1. Create a new wxSQLite database from SQLiteStudio, use the same password as you used for your problematic database.
2. Open it and close it within SQLiteStudio (to make sure file is created).
3. Keep SQLiteStudio running.
4. Replace actual file created with your wxSqlite file
5. Then try to open in in SQLiteStudio again - it may print the error message at this moment. If not, proceed to next step.
6. If 5 step worked (no error printed) you either should see your actual database (let me know if you do), or you probably see empty database, as it failed to read schema. If the latter is the case then:
7. Try to create table in this database. When you try to "commit" new table, it should print actual error.

As for additional logging - yes, you can enable it from command line using -d option. If you use Windows, you will be able to access these logs through debug console (it's in Help menu, or through F12 key). Unfortunately there is not much regrding DB opening issues.
